src/LessonSixTasks.java

```diff
@@ -0,0 +1,84 @@
+package src;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class LessonSixTasks {
+
+    public static void main(String[] args) {
+        int[] array = fillArray();
+        System.out.println(Arrays.toString(array));
+        sortShaker(array);
+
+        int[] sourceArray = {1, 2, 3, 4};
+        int[] targetArray = {5, 6, 7};
+        arrayCopy(sourceArray, targetArray);
+    }
+
+    /**
+     * Task #1
+     */
+    public static int[] arrayCopy(int[] sourceArray, int[] targetArray) {
+
+        int copyLength = sourceArray.length + targetArray.length;
+
+        int[] resultArray = new int[copyLength];
+
+        if (targetArray.length == 0) { // []
+            for (int i = 0; i < copyLength; i++) {
+                resultArray[i] = sourceArray[i];
+            }
+        } else {
+            for (int i = 0; i < targetArray.length; i++) { //
+                resultArray[i] = targetArray[i];
+            }
+
+            for (int i = 0; i < sourceArray.length; i++) {
+                resultArray[targetArray.length + i] = sourceArray[i];
+            }
+        }
+        System.out.println(Arrays.toString(resultArray));
+        return resultArray;
+    }
+
+    /**
+     * Task #2
+     */
+    public static int[] fillArray() {
+        Random random = new Random();
+
+        int[] array = new int[10];
+
+        for (int i = 0; i < array.length; i++) {
+            array[i] = random.nextInt(0, 100);
+        }
+        return array;
+    }
+
+    public static int[] sortShaker(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            boolean isSorted = false;
```

Comment:
Wrong variable name. It is not sorted when two elements are changed. Possibly, wasNotSorted or changePerformed or something else